doc/Command_Index_internal.tex

```diff
@@ -345,6 +353,400 @@ \section{Gregorio\TeX{} Controls}
   \#4 & dimension   & the amount to shift the bar to the left from the end of the base symbol.
 \end{argtable}
 
+\macroname{\textbackslash gre@additionalbottomcustolineend}{}{gregoriotex-signs.tex}
+Macro to place a bottom custo with an additional line (positions \texttt{a} and \texttt{b}) at the end of a line.
```

Comment:
"Custo" in all these statements should really be "custos".  "Custos" is a Latin word meaning guard.  No declension of "custos" is "custo".